src/lib/docker/dockerService.ts

```diff
@@ -347,6 +349,24 @@ class DockerService implements DockerLibrary {
       }
     }
   }
+
+  /**
+   * Renames the docker volume directory on disk when a node is renamed
+   * @param network the network containing the node
+   * @param node the node that's to be renamed
+   * @param newName the new name for the node and directory
+   */
+  async updateDirs(network: Network, node: CommonNode, newName: string) {
+    const volumeDir = (node as AnyNode).implementation
+      .toLocaleLowerCase()
+      .replace('-', '');
+    const oldPath = join(network.path, 'volumes', volumeDir, node.name);
+    const newPath = join(network.path, 'volumes', volumeDir, newName);
+
+    fs.rename(oldPath, newPath)
+      .then(() => console.log(`Renamed directory from ${oldPath} to ${newPath}`))
+      .catch((err: any) => console.error(`Error renaming directory: ${err}`));
```

Comment:
1. This will not block execution while the dir is being renamed. So you may end up in a situation where the docker network is starting up while the rename is still processing. Use `await` instead of `.then` to wait for the rename to complete before proceeding.
2. We have [utils/files.ts](https://github.com/jamaljsr/polar/blob/master/src/utils/files.ts) which contains functions for file operations. It would be better to create a rename function there. I also prefer to use `fs-extra` for cleaner `async` operations.